src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -8025,15 +8025,9 @@ internal static bool CreateJunction(string path, string target)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods")]
         private static bool WinCreateJunction(string path, string target)
         {
-            if (string.IsNullOrEmpty(path))
-            {
-                throw new ArgumentNullException(nameof(path));
-            }
+            ArgumentException.ThrowIfNullOrEmpty(path);
```

Comment:
Please fix test_FileSystemProvider.cs:line 87

Reply:
Is this the right fix? `Assert.Throws<System.ArgumentNullException>`  -> `Assert.Throws<System.ArgumentException>`

Reply:
Yes, it is right.